test/e2e/specs/__snapshots__/adding-blocks.test.js.snap

```diff
@@ -1,21 +1,25 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`adding blocks Should insert content using the placeholder and the regular inserter 1`] = `
-"<!-- wp:paragraph -->
+"<!-- wp:image -->
+<figure class=\\"wp-block-image\\"><img alt=\\"\\" /></figure>
+<!-- /wp:image -->
+
+<!-- wp:paragraph -->
 <p>Paragraph block</p>
 <!-- /wp:paragraph -->
 
+<!-- wp:code -->
+<pre class=\\"wp-block-code\\"><code>Code block</code></pre>
+<!-- /wp:code -->
+
 <!-- wp:paragraph -->
 <p>Second paragraph</p>
 <!-- /wp:paragraph -->
 
 <!-- wp:quote -->
 <blockquote class=\\"wp-block-quote\\">
 	<p>Quote block</p>
-</blockquote>
-<!-- /wp:quote -->
-
-<!-- wp:code -->
-<pre class=\\"wp-block-code\\"><code>Code block</code></pre>
-<!-- /wp:code -->"
+	<p>Can have multiple paragraphs</p><cite>CiteCan have line breaks as well<br/></cite></blockquote>
```

Comment:
It's weird that the `<br>` is not between "Cite" and the rest. Is this a test issue only or a real issue.

Reply:
>It's weird that the <br> is not between "Cite" and the rest. Is this a test issue only or a real issue.

 Hmm I totally missed this. Will look into it.